src/mono/System.Private.CoreLib/src/System/RuntimeFieldHandle.cs

```diff
@@ -72,7 +72,9 @@ public override int GetHashCode()
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         private static extern void SetValueInternal(FieldInfo fi, object? obj, object? value);
 
-        internal static void SetValue(RuntimeFieldInfo field, object? obj, object? value, RuntimeType? _, FieldAttributes _1, RuntimeType? _2, ref bool _3)
+#pragma warning disable IDE0060 // Remove unused parameter (isInitialized)
+        internal static void SetValue(RuntimeFieldInfo field, object? obj, object? value, RuntimeType? _, FieldAttributes _1, RuntimeType? _2, bool isInitialized)
```

Comment:
I would either use `_` for all argument; or use actual names + warning disable for all arguments. This has a mix of both styles now.

Reply:
I reverted that commit and didn't have to modify this area afterwards.